test/parse-result.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * Tests for parse result namespace elements, including all their
+ * convenience properties and methods.
+ */
+
+import chai, {Assertion, expect} from 'chai';
+
+import minim from 'minim';
+import minimApiDescription from 'minim-api-description';
+import minimParseResult from '../src/parse-result';
+
+const namespace = minim.namespace()
+  .use(minimApiDescription)
+  .use(minimParseResult);
```

Comment:
So this is something I've been racking my brain about today and I'm glad you asked it. There are really two questions here:

1. Is the API Description namespace required for the module to work?
2. Is the API Description namespace required for testing?

In this specific case, I believe the answers are 1:not really and 2:yes. The namespace is required during testing because of the convenience functionality for accessing such elements. However, a parse result could contain only annotations or a subsequent namespace could modify the parse result to contain other elements not in the API Description namespace. That's why I say not really to number one.

So, since the namespace has specific requirements on the API Description namespace does that mean it should directly depend on it or not? What about later down the line where you can envision two namespaces that both depend on a third? Should the user be able to compose the tree as they see fit or should modules load their dependencies (possibly loading them multiple times and getting weird behavior)? Thoughts? What about dependency handling?